modules/core/client/services/photos.service.js

```diff
@@ -0,0 +1,176 @@
+    'file': 'an-wavewatching.jpg',
+    'file_mobile': 'an-wavewatching--mobile.jpg'
+  },
+  'hitchgirl1': {
+    // https://www.facebook.com/just.kozmic.blues
+    // Permission granted for Trustroots (asked by Mikael Korpela)
+    'name': 'Gytė',
+    'url': 'https://www.facebook.com/upe.pati.teka',
+    'file': 'gh-hitchgirl1.jpg',
+    'file_mobile': 'gh-hitchgirl1--mobile.jpg'
+  },
+  'hitchgirl2': {
+    // https://www.facebook.com/matasirastrauskas
+    // Permission granted for Trustroots (asked by Mikael Korpela)
+    'name': 'Matas Astrauskas',
+    'url': 'http://www.matasphoto.com/',
+    'file': 'mr-hitchgirl2.jpg'
+  },
+  'mountainforest': {
+    // https://unsplash.com/photos/VNseEaTt9w4
+    'name': 'Sven Scheuermeier',
+    'url': 'https://unsplash.com/sveninho',
+    'license': 'CC',
+    'license_url': 'https://creativecommons.org/publicdomain/zero/1.0/', // https://unsplash.com/license
+    'file': 'ss-mountainforest.jpg',
+    'file_mobile': 'ss-mountainforest--mobile.jpg'
+  },
+  'tribes-1': {
+    // Permission granted for Trustroots (asked by Mikael Korpela)
+    'name': 'Antonio Fulghieri',
+    'url': 'https://aaoutthere.wordpress.com/',
+    'license': 'CC',
+    'license_url': 'https://creativecommons.org/licenses/by-nc-nd/4.0/',
+    'file': 'tribes-1.jpg',
+    'file_mobile': 'tribes-1--mobile.jpg'
+  },
+  'woman-bridge': {
+    'name': 'Michael Hull',
+    'url': 'https://unsplash.com/photos/JibIPPrvITE',
+    'license': 'CC',
+    'license_url': 'https://unsplash.com/license',
+    'file': 'mh-woman-bridge.jpg',
+    'file_mobile': 'mh-woman-bridge--mobile.jpg'
+  },
+  'nordiclights': {
+    'name': 'Vincent Guth',
+    'url': 'https://unsplash.com/photos/62V7ntlKgL8',
+    'license': 'CC',
+    'license_url': 'https://unsplash.com/license',
+    'file': 'nordic-lights.jpg',
+    'file_mobile': 'nordic-lights--mobile.jpg'
+  },
+  'jungleroad': {
+    'name': 'Dean Johns',
+    'license': 'CC',
+    'license_url': 'https://unsplash.com/license',
+    'file': 'jungleroad.jpg',
+    'file_mobile': 'jungleroad--mobile.jpg'
+  },
+  'sahara-backpacker': {
+    'name': 'Nova Togatorop',
+    'url': 'http://novatogatorop.com/',
+    'license': 'CC',
+    'license_url': 'https://unsplash.com/license',
+    'file': 'ak-sahara.jpg',
+    'file_mobile': 'ak-sahara--mobile.jpg'
+  },
+  'hitchtruck': {
+    'name': 'Nova Togatorop',
+    'url': 'http://novatogatorop.com/',
+    'license': 'CC',
+    'license_url': 'https://unsplash.com/license',
+    'file': 'nt-hitchtruck.jpg',
+    'file_mobile': 'nt-hitchtruck--mobile.jpg'
+  }
+};
+
+function getApplicationScope() {
+  return angular.element(document.querySelector('html')).scope();
+}
+
+export function selectPhoto(name) {
+  const photo = photos[name];
+  const file = (window.innerWidth <= 480 && photo.file_mobile) ? photo.file_mobile : photo.file;
+  const imageUrl = `/img/board/${file}`;
+  getApplicationScope().$emit('photoCreditsUpdated', { [name]: photo });
```

Comment:
What is the motivation for having this @nicksellen? It doesn't seem to be required for the application to function and the application fails when it's run with `start:prod` script.
(I also changed the ng-annotate-loader in `config/webpack/webpack.config.js` to babel-plugin-angularjs-annotate, so maybe these two fail when together.)

Reply:
I changed this in 68dece5. To see the original, see the branch at 5efdfa6.